docs/dev/run_the_etl.rst

```diff
@@ -0,0 +1,88 @@
+.. _run-the-etl:
+
+=======================
+Walk, Don't Run the ETL
+=======================
+
+So you want to run the PUDL data processing pipeline? This is the most involved
+way to get access to PUDL data. It is only suggested if you want to edit the
+ETL process or contribute to the code base. Check the basic usage page for the
+other access methods (see: :ref:`basic-usage`). Measure twice, cut once.
```

Comment:
What does the "Measure twice, cut once" mean here?

Reply:
why do hate fun?!?

Reply:
Because it is confusing! I honestly don't know what you mean here, what the analogy you're trying to make is. If there's information content there that you're trying to get across, it should be clear. If there isn't information content, then someone trying to figure out what it means is wasting their time.

Reply:
Here's an analogy: you know how shoveling the snow on your sidewalk is annoying, but absolutely necessary if folks in wheelchairs or other mobility devices are going to be able to use the sidewalk in the winter? Imagine a graduate student who is a non-native English speaker trying to read our documentation. The actual content is complex enough on its own.